backend/typescript/types.ts

```diff
@@ -1,23 +1,23 @@
-export type Role = "User" | "Admin";
+export type Role = "Admin" | "Member" | "Author";
 
 export type Token = {
   accessToken: string;
   refreshToken: string;
 };
 
 export type UserDTO = {
-  id: string;
+  auth_id: string;
```

Comment:
This should be authId and roleType on the types.ts file. Sorry I know it is confusing but basically the convention is on js use camelCase but on postgres use snake case so that's why your model and migration files have snake case